app/services/time-zone.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+const {
+    Service,
+    computed,
+    inject: {service}
+} = Ember;
+
+export default Service.extend({
+    store: service(),
+
+    _parseTimezones(settings) {
+        let activeTimezone = settings.get('activeTimezone');
+        let offset = activeTimezone;
+
+        return offset;
+    },
+
+    _settings: computed(function () {
+        let store = this.get('store');
+        return store.queryRecord('setting', {type: 'blog'});
```

Comment:
This will need to be updated to query `{type: 'blog,theme,private'}` so that we don't run the risk of other areas of the admin saving incomplete settings data.

I'd like to make this problem go away by creating a `settings` service which handles loading/refreshing/saving the blog settings but that's something for a different PR 😄